### DISCUSSION TITLE
Franka Panda object tracking/moveTo

### DISCUSSION THREAD

**Discussion by nikkijhart** [empriselab/RCareWorld#12] [Help - Programming]
When the Franka is not positioned with position [0,0,0] and rotation [0,90,0], the Franka seems to not be able to track objects properly; it moves as they move, but its end effector is not at the location of the object it's tracking. Are there workarounds where the robot could be positioned differently?
![image](https://github.com/empriselab/RCareWorld/assets/125599638/9d6fc883-1316-457c-9de7-1646a0419a0c)

**Comment by YoruCathy** [comment]
For position, have you tried putting a parameter for position when you initialize the robot in Python?

**Reply by nikkijhart** [reply]
Yes, I updated the position parameter when initializing the robot.
